examples/cookie-game/src/state_types.ts

```diff
@@ -0,0 +1,132 @@
+import { actions } from "./actions"
```

Comment:
This file should be renamed to `types`. Maybe inside a `state` package, with `state.ts`. What do you think?